plugin/src/main/java/io/jenkins/plugins/casc/Configurator.java

```diff
@@ -158,4 +158,10 @@ default Class getImplementedAPI() {
     @CheckForNull
     CNode describe(T instance, ConfigurationContext context) throws Exception;
 
+    /**
+     * Jelly hack used to distinguish RootElementConfigurator from other Configurators
+     */
+    default boolean isRootElement() {
+        return this instanceof RootElementConfigurator;
```

Comment:
hacks are okay when they are this useful!

Reply:
was for another fix in progress, I mixed my branches.